pyproject.toml

```
@@ -0,0 +1,5 @@
+[tool.ruff]
+line-length = 115
```

Curious what you think of this:

```suggestion
line-length = 79  # Preferred line length to wrap lines

[tool.ruff.lint.pycodestyle]
max-line-length = 115  # Line length to allow for line-too-long violations
```

I noticed that Ruff will not reformat strings if they exceed `line-length`.

---

79 is probably fine.

> I noticed that Ruff will not reformat strings if they exceed line-length.

Weird